pkgs/development/python-modules/consul2/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi
+, requests, six, pytest }:
+
+buildPythonPackage rec {
+  pname = "python-consul2";
+  version = "0.1.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-KchZ3nPhfzarmb6DH9wdkk8W6HciMyd6KO+St7mZlc0=";
+  };
+
+  buildInputs = [ requests six pytest ];
+
+  # No tests distributed. https://github.com/cablehead/python-consul/issues/133
```

Comment:
Have you tried pulling the source from github instead?

Reply:
Well, I copied how `python-consul` is done, and that package uses `fetchPypi`:
https://github.com/NixOS/nixpkgs/blob/8d85443a6577d9c33e66d28f3121341612a405cb/pkgs/development/python-modules/consul/default.nix#L8-L11
If it's preferable to use `fetchFromGitHub` I can certainly do that.

Reply:
Indeed, using `fetchFromGitHub` does include tests. But I am having trouble making them work, despite adding `twisted` to `checkInputs` list:
```
       > WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff5945de0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/pytest-twisted/
       > ERROR: Could not find a version that satisfies the requirement pytest-twisted (from versions: none)
       > ERROR: No matching distribution found for pytest-twisted
```

Reply:
It's looking for the package `pytest-twisted` rather than `twisted` itself, and you could always substitute the included binaries for symlinks to our `consul` package in the `preCheck`.

I'm keen on tests because without them this package is liable to get silently broken by auto-updates.